src/classes/ADDR_HouseholdAcc_TEST.cls

```diff
@@ -1851,9 +1851,16 @@ public with sharing class ADDR_HouseholdAcc_TEST {
         Test.stopTest();
         
         // verify the Address objects split street.
-        List<Address__c> listAddr = [select Id, MailingStreet__c, MailingStreet2__c, Formula_MailingStreetAddress__c from Address__c 
-        where MailingStreet__c = 'new street'];
+        List<Address__c> addresses = [SELECT Id, MailingStreet__c, MailingStreet2__c, Formula_MailingStreetAddress__c from Address__c];
```

Comment:
Curious about this. Why can't we loop through addresses and update the mailing street field?

Reply:
We could do an update here I suppose, but its more DML. This way I could just query all addresses and simply filter out the ones we care about, in this case 'new street'

I think that's what you're asking...if I'm wrong lemme know.